### PR TITLE
Add a new TypeSpec Feed using autorest-canonical emitter

### PR DESCRIPTION
**Pull request by dinwa-ms**

### CODE REVIEW

File: eng/feeds/arm-canonical/tspconfig_stand_alone.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+emit:
```
Comment (markcowl) [inline eng/feeds/arm-canonical/tspconfig_stand_alone.yaml:1]: do not need this

File: eng/feeds/azure-canonical-scaffolding.json
[the remark this answers is not in the document]
Comment (markcowl) [reply eng/feeds/azure-canonical-scaffolding.json:1]: @timotheeguerin  This is limited to only a few teams for this Semester, and the idea was not to confuse general users with this.  Once this idea is proven, I think we would either replace the existing template or add this as an option in the standard feed.

File: eng/feeds/azure-canonical-scaffolding.json [hunk 2]
```diff
@@ -0,0 +1,308 @@
+{
+  "azure-core": {
```
Comment (markcowl) [inline eng/feeds/azure-canonical-scaffolding.json:2]: should only need your template

File: eng/feeds/azure-canonical-scaffolding.json [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+    ],
+    "inputs": {
+      "ServiceNamespace": {
+        "description": "Please provide service namespace in Pascal case:",
+        "type": "text",
+        "initialValue": "Azure.Contoso"
+      }
+    }
+  },
+  "azure-arm": {
```
Comment (markcowl) [inline eng/feeds/azure-canonical-scaffolding.json:56]: do not need this

File: eng/feeds/azure-canonical-scaffolding.json [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+    ],
+    "inputs": {
+      "ServiceNamespace": {
+        "description": "Please provide ARM Resource Provider Name in Pascal case:",
+        "type": "text",
+        "initialValue": "Microsoft.Contoso"
+      }
+    }
+  },
+  "azure-arm-canonical": {
```
Comment (markcowl) [inline eng/feeds/azure-canonical-scaffolding.json:120]: this is the only one that is needed

File: eng/feeds/azure-canonical-scaffolding.json [hunk 5]
```diff
@@ -0,0 +1,308 @@
...
+    ],
+    "inputs": {
+      "ServiceNamespace": {
+        "description": "Please provide ARM Resource Provider Name in Pascal case:",
+        "type": "text",
+        "initialValue": "Microsoft.Contoso"
+      }
+    }
+  },
+  "azure-core_stand_alone": {
```
Comment (markcowl) [inline eng/feeds/azure-canonical-scaffolding.json:184]: do not need this

File: eng/feeds/azure-canonical-scaffolding.json [hunk 6]
```diff
@@ -0,0 +1,308 @@
...
+    ],
+    "inputs": {
+      "ServiceNamespace": {
+        "description": "Please provide service namespace in Pascal case:",
+        "type": "text",
+        "initialValue": "Azure.Contoso"
+      }
+    }
+  },
+  "azure-arm_stand_alone": {
```
Comment (markcowl) [inline eng/feeds/azure-canonical-scaffolding.json:241]: don't need this

**Review by markcowl — CHANGES_REQUESTED** [review]

File: eng/feeds/azure-canonical-scaffolding.json (hunk 6 above)
Comment (dinwa-ms) [reply eng/feeds/azure-canonical-scaffolding.json]: removed

File: eng/feeds/azure-canonical-scaffolding.json (hunk 5 above)
Comment (dinwa-ms) [reply eng/feeds/azure-canonical-scaffolding.json]: removed

File: eng/feeds/azure-canonical-scaffolding.json (hunk 3 above)
Comment (dinwa-ms) [reply eng/feeds/azure-canonical-scaffolding.json]: removed

File: eng/feeds/azure-canonical-scaffolding.json (hunk 2 above)
Comment (dinwa-ms) [reply eng/feeds/azure-canonical-scaffolding.json]: removed

File: eng/feeds/arm-canonical/tspconfig_stand_alone.yaml (hunk 1 above)
Comment (dinwa-ms) [reply eng/feeds/arm-canonical/tspconfig_stand_alone.yaml]: removed

**Review by markcowl — APPROVED** [review]

File: eng/feeds/azure-canonical-scaffolding.json [hunk 7]
```diff
@@ -0,0 +1,66 @@
+{
+  "azure-arm-canonical": {
+    "title": "(rest-api-spec repo) Azure Resource Manager Service Project with Canonical Swagger",
+    "description": "Create a project with canonical swagger in rest-api-spec repo, representing an Azure service ARM API",
```
Comment (allenjzhang) [inline eng/feeds/azure-canonical-scaffolding.json:4]: `Create standard ARM API project with an additional canonical swagger emitter in rest-api-spec repo`

Comment (allenjzhang) [reply eng/feeds/azure-canonical-scaffolding.json:4]: Just noticed it is separate scaffolding json. Above not neccessary.

**Pull request merged**